youtube_dl/extractor/youtube.py

```diff
@@ -12,7 +12,7 @@
 import traceback
 
 from .common import InfoExtractor, SearchInfoExtractor
-from ..jsinterp import JSInterpreter
+from ..jsinterp2 import JSInterpreter
```

Comment:
Accidental changes, but oddly passed `test_youtube_signature`. For me it fails due to `JSArrayPrototype._slice` doesn't handle arguments correctly since not being implemented yet.